.github/workflows/main.yml

```diff
@@ -0,0 +1,48 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
```

Comment:
You don't have to change this, especially given its the only workflow, but if you add more you might want to make this more descriptive, as it ends up as the top level name in the left menu under "Actions".